core/src/main/scala/chisel3/internal/BiConnect.scala

```diff
@@ -180,15 +180,15 @@ private[chisel3] object BiConnect {
         ) {
           pushCommand(Connect(sourceInfo, leftReified.get.lref, rightReified.get.lref))
         } else if (!emitStrictConnects) {
```

Comment:
should we change `emitStrictConnects` to `emitFirrtlConnects`?

Reply:
Isn't that the name of the compile option you made?

Reply:
yeah I am asking if we should change it

Reply:
we can do in  a follow-up